ci.cue

```diff
@@ -355,5 +360,24 @@ dagger.#Plan & {
 				}
 			}
 		}
+		aiops: {
```

Comment:
is it the right way to do? looks like it builds everything under `aiops` but don't we only want the open search-update-service

Reply:
This was just sort of an example for the new code. It is not replacing the current system yet.